modules/geoip/geoip_main.cpp

```diff
@@ -14,16 +14,19 @@
 #include "geoip_main.h"
 #include "geoip_natives.h"
 #include "geoip_util.h"
-#include <time.h>
+#include <ctime>
```

Comment:
It is not required that cheader definitions are available in global namespace.
http://eel.is/c++draft/headers#4
http://eel.is/c++draft/headers#footnote-172
http://eel.is/c++draft/contents#footnote-166

Reply:
You always comment only when things are merged.

I believe this change has been made by my editor, probably because those headers are deprecated in c++. In our context, I guess time.h should be kept